TLM/TLM/UI/UIBase.cs

```diff
@@ -105,6 +105,7 @@ public class UIBase : UICustomControl {
 
             GetMenu().Show();
             LoadingExtension.TranslationDatabase.ReloadTutorialTranslations();
+            LoadingExtension.TranslationDatabase.ReloadGuideTranslations();
```

Comment:
There's a duplicate of these lines in `Options.cs` - would it be worth having a central method for when language changes? Not sure where it would go however.